webpack/components/AnsibleRolesAndVariables/AnsibleRolesAndVariables.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  cellWidth,
+} from '@patternfly/react-table';
+import {
+  Button,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarContent,
+  ToolbarItem,
+  Checkbox,
+  Pagination,
+} from '@patternfly/react-core';
+
+import PropTypes from 'prop-types';
+import { DEFAULT_PER_PAGE } from './AnsibleRolesAndVariablesConstants';
+
+const ImportRolesAndVariablesTable = ({
+  columnsData,
+  rowsData,
+  proxy,
+  onSubmit,
+  onCancel,
+}) => {
+  const columns = columnsData.map(col => ({
+    ...col,
+    transforms: [cellWidth(10)],
+  }));
+  const [rows, setRows] = useState(rowsData);
```

Comment:
Not sure if you are aware that initializing state from props has pitfalls - `useState` does not update the state when prop value changes. No problems are manifested here since props are passed from erb on mount, but they would be apparent when loading data asynchronously.